charts/portworx/templates/hooks/pre-install/etcd-preinstall-hook.yaml

```diff
@@ -38,9 +38,9 @@ spec:
         imagePullPolicy: Always
 
         {{- if eq $customRegistryURL "none" }}
-        image: "hrishi/px-etcd-preinstall-hook:v2"
+        image: "portworx/px-etcd-preinstall-hook:v1"
```

Comment:
Add a travis job in this repo and builds the docker image for portworx/px-etcd-preinstall-hook. For the master branch, always push to a :latest tag. We will manually release v1 from latest.

All the environment variables for travis need to be stored in the travis job settings.

For examples on a travis job, see https://github.com/portworx/torpedo/blob/master/.travis.yml

Reply:
@harsh-px. This has been fixed. 

A run with the travis build. 
https://travis-ci.org/portworx/px-preinstall-hook/builds/487306646